engine-extensions/src/main/java/io/nosqlbench/engine/extensions/computefunctions/ComputeFunctions.java

```diff
@@ -43,6 +45,7 @@
  * elide duplicates internally.
  */
 public class ComputeFunctions extends NBBaseComponent {
+    private final static Logger logger = LogManager.getLogger("RUNTIME");
```

Comment:
This should not be "RUNTIME" as this was intended to be for component aspects specifically. (component tree changes, eventing, etc)

Reply:
removed logger in latest update, WARN replaced with appropriate Exceptions